runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabInterface.java

```diff
@@ -186,11 +188,30 @@ private TabInterface(
 		this.searchProvider = searchProvider;
 	}
 
+	private static MenuEntry[] createMenuEntry(MenuEntryAdded event, String option, String target, MenuEntry[] entries)
```

Comment:
Why did this move

Reply:
Automatic formatter moved it. I moved it to it's original location now.